### PR TITLE
Feature/salah/add pixel and snapchat

### PR DESCRIPTION
**Pull request by Salomari1987**
PR [#20](https://github.com/Edraak/edraak-2015-theme/pull/20) on themes
PR [#169](https://github.com/Edraak/marketing-site/pull/169) on marketing

### CODE REVIEW

File: lms/static/js/student_account/views/RegisterView.js [hunk 1]
```diff
@@ -92,7 +94,8 @@
                     eventValue: 0,
                     transport: 'beacon'
                 });
-
+                fbq('track', 'CompleteRegistration', {status: 'failed'});
+                snaptr('track', 'SIGN_UP', {success: 0});
```
Comment (devalih) [inline lms/static/js/student_account/views/RegisterView.js]: I don't like that you're defining the script in the theme repo and use it here, let us talk.

Comment (devalih) [reply lms/static/js/student_account/views/RegisterView.js]: Or basically this needs to be controlled by configs

Comment (Salomari1987) [reply lms/static/js/student_account/views/RegisterView.js]: I see, the way the theme is built I cannot run this from within the theme, and according to edX we should add the scripts for analytics in theme. However, we need to do a check here if these objects exist.

**Comment by Salomari1987** [comment]
@devalih nudge

**Pull request merged**